### DISCUSSION TITLE
unconstrained delegation set error

### DISCUSSION THREAD

**Discussion by ghost** [Marshall-Hallenbeck/red_team_attack_lab#17] [Q&A]
This is our environnement lab.
Linux HP-EliteDesk-800-G4-TWR 5.13.0-22-generic #22-Ubuntu SMP Fri Nov 5 13:21:36 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
We have an error with unconstrained user
![Capture d’écran du 2021-12-22 15-52-27](https://user-images.githubusercontent.com/88156465/147116989-1a16c267-c919-4874-9aa5-e45e7eea714a.png)
What could be a solution to solve this problem .Thank you

**Comment by Marshall-Hallenbeck** [comment]
Unfortunately random errors like this happen all the time and I'm not 100% sure why. I think most of them are just random. You can try rerunning the provision with `vagrant up dc01 --provision` or just destroy it with `vagrant destroy dc01` and reload it with `vagrant up dc01`

**Comment by ghost** [comment]
Ok thank you so much .

**Comment by Marshall-Hallenbeck** [comment]
If this keeps happening let me know, but I just booted up an environment yesterday and it worked, though I did run into a random issue and had to re-run it.
